docs/object.js

```diff
@@ -46,4 +46,11 @@ class Object {
      * @since 1.9.0
      */
     linkingObjects(objectType, property) {}
+
+    /**
+     * Returns the total count of incoming links to this object
+     * @returns {number} number of links to this object.
+     * @since 2.6.0
+     */
+    linkingObjectsCount(objectType, property) {}
```

Comment:
The parameter should be documented too. They can be copied from the method above.

Reply:
Yes, it should not take any parameters. It was a copy/paste mistake. Fixed.